cfy_manager/components/restservice/config/haproxy.cfg

```diff
@@ -18,7 +18,7 @@ listen postgres
     option httpchk
     http-check expect status 200
     default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
-{%- for node in postgresql_server.cluster.nodes %}
-    server postgresql_{{ node }}_5432 {{ node }}:5432 maxconn 100 check check-ssl port 8008 ca-file /etc/haproxy/ca.crt
+{%- for name, node in postgresql_server.cluster.nodes.iteritems() %}
```

Comment:
If we're discarding the name it'd be better to do:
for node in postgresql_server.cluster.nodes.values()